lib/core/utils/is-hidden.js

```diff
@@ -8,7 +8,8 @@
  */
 axe.utils.isHidden = function isHidden(el, recursed) {
 	'use strict';
-	var parent;
+	const node = axe.utils.getNodeFromTree(el);
+	let parent, isHidden;
```

Comment:
Slight preference for declaring variables inline rather than at the top of the function. That way you can use const for both of these.